### ISSUE TITLE
Controls are overlapping and not visible after resizing the page to 200%.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#185]
![image](https://user-images.githubusercontent.com/2320302/119194009-51023980-ba37-11eb-8514-1abebae5bf45.png)
